eng/AndroidX.targets

```diff
@@ -61,5 +61,10 @@
         Update="Xamarin.AndroidX.Window.WindowJava"
         Version="1.0.0.9"
     />
+	<PackageReference
+        Update="Xamarin.AndroidX.WebKit"
+        Version="1.4.0.8"
+    />
```

Comment:
If this is an `Update`, how does this line do anything? It must already be a transitive dependency, and we can remove it?

Reply:
@[javiercn](https://github.com/javiercn) would you mind filing an issue first explaining the motivation for the change as well as the impact so that the team can assess whether or not we want to take this change in?
>
Please forgive me for not having time to reply a few days ago.

This is not necessary, but just a suggestion for the following reasons.

1. From Android API 24 to 23, it is not difficult for Android. They are all official.

2. API 21 app will run on approximately 98.6% of devices, and API 24 app only has 91.7%. The is from Android studio.

3. Using androidx.webkit increases the size of Android installation package a little and the performance loss is almost negligible.

4. It makes the Android device compatibility of Maui Blazor consistent with Maui.

5. My test method is temporarily modified the namespace of Microsoft.AspNetCore.Components.WebView.Maui.csproj and only compile it with min Android API 21, so that it can be directly added to the Maui blazor app project, and then tested on the real machine of Android 5(API 21). The startup speed is acceptable, which is 40% faster than blazor WebAssembly. This is stupid, but simple and effective.